modules/src/core/ics04_channel/handler/chan_open_try.rs

```diff
@@ -22,7 +22,7 @@ pub(crate) fn process(
     // Unwrap the old channel end (if any) and validate it against the message.
     let (mut new_channel_end, channel_id) = match &msg.previous_channel_id {
```

Comment:
If `ChannelId` is `Copy`, would it be nicer to match it by value and avoid the dereferencing cruft in match arms?

Reply:
Do we always want to pass `ChannelId` by value? Halfway through this and the diff is huge, wondering if it's worth it ->
```
$ git diff --shortstat
 49 files changed, 220 insertions(+), 220 deletions(-)
```

Reply:
It's more idiomatic but perhaps not worth it indeed. Let's file it in the backlog :)

Reply:
Added this as an entry to this meta issue https://github.com/informalsystems/ibc-rs/issues/1696